test/profiler/test_profiler.py

```diff
@@ -664,6 +664,9 @@ def fn(nt):
             for i in range(3):
                 values = torch.rand((8 + i, 4 + i))
                 offsets = torch.tensor([0, 2, 4, 6, 8 + i])
+                # Need to explicitly graph break due to:
+                # https://github.com/pytorch/pytorch/issues/124500
+                torch._dynamo.graph_break()
```

Comment:
instead can we just `@skipIfTorchDynamo`? I don't think we really need to support torch.compile-ing a function that includes a profiler/execution trace